match/lib/match/nuke.rb

```diff
@@ -125,8 +135,10 @@ def prepare_list
       # Get all iOS and macOS profile
       self.profiles = []
       prov_types.each do |prov_type|
-        self.profiles += profile_type(prov_type).all(mac: false)
-        self.profiles += profile_type(prov_type).all(mac: true)
+        types = profile_types(prov_type)
+        # Filtering on 'profileType' seems to be undocumented as of 2020-07-30
+        # but works on both web session and official API
```

Comment:
Is this still the case as of today? Maybe it's gotten better since then 🤞 ? 😁

Reply:
Still the case 😔